Renderer.js

```diff
@@ -0,0 +1,89 @@
+class Renderer
+{
+    constructor()
+    {
+        this.data = {};
+    }
+
+    renderData(data)
+    {
+        new RenderUsersData(data.users);
+        new RenderQuotesData(data.quotes);
+        new RenderBaconsData(data.bacons);
+        new RenderPokemonsData(data.pokemons);
```

Comment:
again, no need for all these unnecessary classes.... just have one Render class that takes care of all the rendering